releases/2017_cycle_3.md

```diff
@@ -0,0 +1,46 @@
+[countdown to release](https://www.timeanddate.com/countdown/to?iso=20170905T09&p0=179&msg=2017+Cycle+3+Release&ud=1&font=sanserif&csz=1)
+
+Release name: 2017 Cycle 3
+
+Release date: Sept 5th (First day of cycle)
+Target Audience: 
+- Eric Dooryhee, Sanjit Ghose and Milinda Abeykoon
+
+Scope: 
+- Analysis/visualization pipeline
+    - Live data analysis from raw images to PDFs
+    - Live visualization: 
+        - Dark/Background corrected images
+        - Generated Mask
+        - Waterfall plot of I(Q)
+        - Waterfall plot of F(Q)
+        - Waterfall plot of G(r)
+    - Automated file writing
+        - Dark corrected tiffs
+        - Masks (in Fit2d format)
+        - I(Q) and I(tth)
+        - G(r)
+- Enhanced Metadata Schema
+    - Stronger links for calibration metadata
+    - More facility level metadata
+    - Removal of `NAN`s from metadata
+- `Tlist` Shutter Protocol
+    - Shutter is optionally now closed when not taking images, saving the detector from burn
+- Long term configuration
+    - Long term configuration file provides beamline level metadata. This will enable searches for data which came from a configuration that matches the current one.
```

Comment:
replace with: 
Long term configuration file provides infrequently varying beamline level metadata, for example the hardware configuration. This will enable searches for data which came from a configuration that matches the current one.